tracer/test/Datadog.Trace.TestHelpers/LogSettingsHelper.cs

```diff
@@ -26,15 +36,8 @@ internal class LogSettingsHelper
 
         public static ImmutableDirectLogSubmissionSettings GetValidSettings()
         {
-            return ImmutableDirectLogSubmissionSettings.Create(
-                host: "some_host",
-                source: "csharp",
-                intakeUrl: "https://localhost:1234",
-                apiKey: "abcdef",
-                minimumLevel: DirectSubmissionLogLevel.Debug,
-                globalTags: new Dictionary<string, string>(),
-                enabledLogShippingIntegrations: ImmutableDirectLogSubmissionSettings.SupportedIntegrations.Select(x => x.ToString()).ToList(),
-                batchingOptions: new BatchingSinkOptions(1000, 100_000, TimeSpan.FromSeconds(2)));
+            var tracerSettings = new TracerSettings(new NameValueConfigurationSource(ValidDefaults));
```

Comment:
Just an FYI, but there's a helper on TracerSettings to make this simpler so you don't need the multiple `NameValueConfigurationSource` etc

```suggestion
            var tracerSettings = TracerSettings.Create(new()
            {
               { ConfigurationKeys.ApiKey, "abcdef" },
               { ConfigurationKeys.DirectLogSubmission.Host, "some_host" },
                // ...
            });
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/5126/commits/7d4b6a84d089ba85674e50047ce60bc18a403cfc